src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/hooks/helpers.js

```diff
@@ -20,6 +20,10 @@ const getEventStatus = (event: ApiTEIEvent) => {
     if (isEventOverdue(event)) {
         return { status: statusTypes.OVERDUE, options: daysUntilDueDate ? dueDateFromNow : undefined };
     }
+    // DHIS2-11576: VISITED status is treated as ACTIVE
+    if (event.status === 'VISITED') {
```

Comment:
Hey, does it make sense to move the `VISITED` string inside the `statusTypes` const as `statusTypes.VISITED`?

Reply:
hey @simonadomnisoru we supposed to remove the status `VISITED` so I think it's better not to include inside statusTypes